src/main/java/com/mraof/minestuck/inventory/captchalogue/CaptchaDeckHandler.java

```diff
@@ -160,19 +161,33 @@ public static void captchalogueItem(ServerPlayerEntity player)
 	
 	public static void captchalogueItemInSlot(ServerPlayerEntity player, int slotIndex, int windowId)
 	{
-		if(canPlayerUseModus(player) && hasModus(player) && player.openContainer.windowId == windowId && player.openContainer.getCanCraft(player))
```

Comment:
The window id check here is meant to verify that we're captchaloguing in the intended container. What's the motivation for removing this?